pkg/dao/types/status/status_workflow_execution.go

```diff
@@ -14,11 +15,15 @@ const (
 //	| Running          | Unknown                 | Running               | Transitioning         |
 //	| Running          | False                   | Failed                | Error                 |
 //	| Running          | True                    | Completed             | Completed             |
+//	| Stopped          | Unknown                 | Stopping              | Transitioning         |
+//	| Stopped          | False                   | Failed                | Error                 |
+//	| Stopped          | True                    | Stopped               | Completed             |
 var workflowExecutionStatusPaths = NewWalker(
 	[][]ConditionType{
 		{
 			WorkflowExecutionStatusPending,
 			WorkflowExecutionStatusRunning,
+			WorkflowExecutionStatusStopped,
```

Comment:
Is it inactive? ref: https://github.com/seal-io/walrus/pull/1737

Reply:
Is it inappropriate to use the `inactive` status to indicate an stopped workflow?

Reply:
How do we show the status in the UI?

Reply:
How do we show the status in the UI?

Reply:
Let's using `Canceled` for stop a workflow.